apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs

```diff
@@ -122,4 +122,248 @@ private Operation PollForCompletion(Operation operation, string alias)
             return operation;
         }
     }
+    [Collection(nameof(ComputeFixture))]
+    public sealed class InstancesTests{
```

Comment:
This should be in a new file called InstancesTest.cs (and called InstancesTest rather than InstancesTests). Alternatively, just leave it as part of AcceptanceTest if we don't particularly need it to be broken out.

Reply:
Moved